### PR TITLE
local storage and reset function

### PR DESCRIPTION
**Pull request by Anita77777**
With this PR I added local storage, the function reset for the input fields and created a new component for the card element.

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+	testEnvironment: 'jest-environment-jsdom',
+	resolver: `${__dirname}/jest-resolver.js`,
 };
+
+/* const { defaults } = require('jest-config');
+module.exports = {
+	...defaults,
+	resolver: `${__dirname}/jest-resolver.js`,
+}; */
+
```
Comment (doemser) [inline jest.config.js]: always get rid of unused code ;)

File: src/components/Card/Card.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+							<p>Author: {book.author}</p>
+							<p>Title: {book.title}</p>
+							<p>{book.comment}</p>
+						</li>
+					</CardContainer>
+				);
+			})}
+		</ul>
+	);
+}
```
Comment (doemser) [inline src/components/Card/Card.js]: Consider renaming your "newBook" state to something that fits better. newBook seems to be an array of books, so why not calling it "books" or "newBooks"?

File: pages/index.js [hunk 3]
```diff
@@ -1,9 +1,16 @@
 import Form from '../src/components/Form/Form';
 
+import dynamic from 'next/dynamic';
+
 export default function Home() {
+	const Card = dynamic(() => import('../src/components/Card/Card'), {
+		ssr: false,
+	});
+
```
Comment (doemser) [inline pages/index.js]: Please add a little comment for that, so everyone seeing this in future, knows why you did that.

**Review by doemser — CHANGES_REQUESTED** [review]
Good Work!

Just a few minor changes I request ;)

File: pages/index.js (hunk 3 above)
Comment (Anita77777) [reply pages/index.js]: All right

File: src/components/Card/Card.js (hunk 2 above)
Comment (Anita77777) [reply src/components/Card/Card.js]: Changed

File: jest.config.js (hunk 1 above)
Comment (Anita77777) [reply jest.config.js]: Is deleted:-)

**Comment by Anita77777** [comment]
Hi Domik, changed as per your request

**Review by doemser — APPROVED** [review]
Yep, well done :)

**Pull request merged**